### DISCUSSION TITLE
add this

### DISCUSSION THREAD

**Discussion by arandomuser0** [winstxnhdw/lc-hax#501] [Ideas]
See where the monster will walk.

**Comment by joep26020** [comment]
> See where the monster will walk.

use imperium its a mod that lets u see stuff like that

**Reply by arandomuser0** [reply]
yeah but its a host only mod

**Reply by winstxnhdw** [reply]
imperium’s codebase is beautiful and its UX is exactly how i’ve always wanted lc-hax to be. unfortunately, i don’t think i’ll have the time to invest porting it soon. maybe during my holidays.
